src/Chamilo/CoreBundle/Component/Editor/CkEditor/Toolbar/SurveyQuestion.php

```diff
@@ -88,7 +89,7 @@ protected function getMinimizedToolbar()
         return [
             $this->getNewPageBlock(),
             ['Undo', 'Redo'],
-            ['Link', 'Image', 'Video', 'Oembed', 'Flash', 'Youtube', 'Audio', 'Table', 'Asciimath', 'Asciisvg'],
+            ['Link', 'Image', 'Video', 'Oembed', 'Flash', 'Youtube', 'VimeoEmbed', 'Audio', 'Table', 'Asciimath', 'Asciisvg'],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 126 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5500355)